lib/puppet/face/config.rb

```diff
@@ -37,7 +37,7 @@
 
   action(:print) do
     summary _("Examine Puppet's current settings.")
-    arguments _("all | <setting> [<setting> ...]")
+    arguments _("all \\| <setting> [<setting> ...]")
```

Comment:
Several of the man pages were broken by the use of unescaped metacharacters. I'm fixing them as I see them to get the man pages building properly.